docs/manual_installation.md

```diff
@@ -78,15 +80,15 @@ pip install -r requirements.txt
 ####
 - finaly start the bot
 ```bash
-./run.sh configs/config.json
```

Comment:
than there should be a second doc for that information not in the manual installation that coffers the manual installation ;)